MSAL/src/MSIDInteractiveRequestParameters+MSALRequest.m

```diff
@@ -41,29 +41,26 @@ - (BOOL)fillWithWebViewParameters:(MSALWebviewParameters *)webParameters
     __typeof__(webParameters.parentViewController) parentViewController = webParameters.parentViewController;
     
 #if TARGET_OS_IPHONE
-    if (@available(iOS 13.0, *))
+    if (parentViewController == nil)
     {
-        if (parentViewController == nil)
-        {
-            NSError *msidError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidDeveloperParameter, @"parentViewController is a required parameter on iOS 13.", nil, nil, nil, nil, nil, YES);
-            if (error) *error = msidError;
-            return NO;
-        }
-        
-        if (parentViewController.view.window == nil)
-        {
-            NSError *msidError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidDeveloperParameter, @"parentViewController has no window! Provide a valid controller with view and window.", nil, nil, nil, nil, nil, YES);
-            if (error) *error = msidError;
-            return NO;
-        }
+        NSError *msidError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidDeveloperParameter, @"parentViewController is a required parameter on iOS 13.", nil, nil, nil, nil, nil, YES);
```

Comment:
Shouldn't this error say `... required parameter on iOS 14` since that's the required minimum now?

Reply:
you are right, thanks . I have it fixed in this PR : https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1630